app/config/wp-demo/download.sh

```diff
@@ -4,12 +4,12 @@
 
 ###############################################################################
 
-[ -z "$CMS_VERSION" ] && CMS_VERSION=4.3
+[ -n "$CMS_VERSION" ] && CMS_VERSION=" --version=$CMS_VERSION"
```

Comment:
The content `4.3` and `--version=4.3` should be in different variables. It hurts my brain when the structure/purpose of a variable changes.

What about:

```php
WPCLI_ARGS=
[ -n "$CMS_VERSION" ] && WPCLI_ARGS="$WPCLI_ARGS --version=$CMS_VERSION"
```